galaxy_importer/loaders.py

```diff
@@ -193,7 +193,7 @@ def _load(self):
 
     def get_requires_ansible(self):
         if not self.data:
-            return
+            raise exc.RuntimeFileError("'requires_ansible' in meta/runtime.yml is mandatory")
```

Comment:
Probably fine. Maybe tweak the error msg to indicate the contents of meta/runtime.yml was empty?

Reply:
updated